src/components/CreateMenu.js

```diff
@@ -51,6 +79,7 @@ class CreateMenu extends PureComponent {
     }
 
     render() {
+        console.debug(this.props.popOverType, this.getAnchorPosition());
```

Comment:
I think this can be removed? I don't think it's very useful to have in production.

Reply:
Sorry for this one, will remove it.